packages/kbn-router-to-openapispec/src/process_versioned_router.test.ts

```diff
@@ -112,8 +60,11 @@ describe('extractVersionedRequestBodies', () => {
 
 describe('extractVersionedResponses', () => {
   test('handles full response config as expected', () => {
-    expect(extractVersionedResponses(route, oasConverter, ['application/test+json'])).toEqual({
+    expect(
+      extractVersionedResponses(createTestRoute(), oasConverter, ['application/test+json'])
+    ).toEqual({
       200: {
+        description: 'OK response 2023-10-31\nOK response 2024-12-31', // merge multiple version descriptions
```

Comment:
For versioned endpoints with more than one version (none right now) we will merge their descriptions instead of letting the last version win. Open to different approaches here.